koswat/cost_report/summary/koswat_summary_location_matrix_builder.py

```diff
@@ -30,11 +28,11 @@ class KoswatSummaryLocationMatrixBuilder(BuilderProtocol):
 
     def _get_multi_location_profile_to_dict_matrix(
         self, locations_profile: MultiLocationProfileCostReport
-    ) -> dict[PointSurroundings, StrategyReinforcementTypeCosts]:
+    ) -> dict[PointSurroundings, StrategyReinforcementType]:
```

Comment:
Don't change names of classes without previous agreement, specially when working in parallel branches.